packages/tpetra/core/src/Tpetra_Import_Util2.hpp

```diff
@@ -333,7 +328,7 @@ sortCrsEntries (const rowptr_array_type& CRS_rowptr,
 {
   // Generate dummy values array
   typedef typename colind_array_type::execution_space execution_space;
-  typedef typename CrsMatrix<>::impl_scalar_type scalar_type;
+  typedef Tpetra::Details::DefaultTypes::scalar_type scalar_type;
```

Comment:
Another option that doesn't depend on the `Details` namespace, would be to use `Tpetra::Vector<>::scalar_type`.  This is just fine for now :-D .